### PR TITLE
Videos page and fixes

### PR DESCRIPTION
**Pull request by pldespaigne**
- [x] add videos page
- [x] small ui fixes

### CODE REVIEW

File: libs/movie/src/lib/movie/pages/additional-information/additional-information.component.html [hunk 1]
```diff
@@ -1,115 +1,119 @@
...
 
-    <section id="box-office" *ngIf="units$ | async as units">
       <h3>Box Office</h3>
+      <section id="box-office" *ngIf="units$ | async as units">
 
-      <bf-form-table [columns]="boxOfficeColumns" [form]="boxOffice" buttonText="Add a Box Office">
-        <ng-template formView let-form>
-          <article [formGroup]="form" fxLayout="column" fxLayoutGap="24px">
+        <bf-form-table [columns]="boxOfficeColumns" [form]="boxOffice" buttonText="Add a Box Office">
+          <ng-template formView let-form>
```
Comment (fritzschoff) [inline libs/movie/src/lib/movie/pages/additional-information/additional-information.component.html:44]: rename the form to somoething else. You got two form vars now. The one from the class and your locally one here. Hard to debug

File: libs/movie/src/lib/movie/pages/additional-information/additional-information.component.html [hunk 2]
```diff
@@ -1,115 +1,119 @@
...
 
-            <div fxLayout fxLayoutGap="24px">
-              <form-country test-id="boxoffice-territory" [form]="form.get('territory')"></form-country>
+              <div fxLayout fxLayoutGap="24px">
+                <form-country test-id="boxoffice-territory" [form]="form.get('territory')"></form-country>
+
+                <mat-form-field appearance="outline">
+                  <mat-label>Metrics</mat-label>
+                  <mat-select test-id="boxoffice-unit" formControlName="unit">
+                    <mat-option *ngFor="let unit of unitBox | keyvalue" [value]="unit.key">
```
Comment (fritzschoff) [inline libs/movie/src/lib/movie/pages/additional-information/additional-information.component.html:53]: could you use the static-select here?

File: libs/movie/src/lib/movie/pages/promotional-elements/media-videos/media-videos.component.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  styleUrls: ['./media-videos.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class MovieFormMediaVideosComponent {
+  form = this.shell.form;
+
+  constructor(
+    private shell: MovieFormShellComponent,
+  ) {
+    this.form.valueChanges.subscribe(a => console.log(a));
```
Comment (fritzschoff) [inline libs/movie/src/lib/movie/pages/promotional-elements/media-videos/media-videos.component.ts:16]: remoooove

Comment (pldespaigne) [reply libs/movie/src/lib/movie/pages/promotional-elements/media-videos/media-videos.component.ts:16]: good catch !

**Pull request merged**